tests/test_monitor.py

```diff
@@ -158,7 +158,7 @@ def test_move_monitor(hlwm):
 
 
 def test_move_monitor_completion(hlwm):
-    assert hlwm.attr.monitors.focus.geometry() in hlwm.complete('move_monitor 0')
+    assert hlwm.get_attr('monitors.focus.geometry') in hlwm.complete('move_monitor 0')
```

Comment:
What's the difference?

Reply:
the completions is always a list of strings, but `hlwm.attr.monitors.focus.geometry()` is now of type `Rectangle`. I've just replaced it with `str(hlwm.attr.monitors.focus.geometry)`, maybe this emphasizes more what's going on.